apps/et/et-sya-api/ithc.yaml

```diff
@@ -9,6 +9,7 @@ spec:
     java:
       image: hmctspublic.azurecr.io/et/sya-api:pr-242-1da6db2-20230414110812 #{"$imagepolicy": "flux-system:ithc-et-sya-api"}
       environment:
+        EXUI_CASE_DETAILS_URL: "https://manage-case.{{ .Values.global.environment }}.platform.hmcts.net/cases/case-details/"
```

Comment:
instead of values.global..., you can set it directly to ithc

Reply:
Thanks, I have updated this